Client/src/screens/ResultScreen/ResultScreen.tsx

```diff
@@ -34,15 +37,37 @@ export function ResultScreen(): React.ReactElement {
   }, []);
 
   return (
-    <div>
+    <div className="start-screen-center">
       <div className="start-screen-center">
         <img
-          src="AwardRibbon.png"
+          src="/AwardRibbon.png"
           alt="Award Ribbon"
           className="careercraft-award-logo"
         />
         <H1>Great work, {userName}!</H1>
-        {finalMessage}
+        <H2>{finalMessage}</H2>
```

Comment:
If you call the backend endpoint /api/career-game/complete the output will now be: {"final_recap": "...","lessons_learned":["...","...","..."]}

Can we format this to be nicely displayed?

Reply:
sure thing.